packages/driver/test/cypress/integration/commands/assertions_spec.js

```diff
@@ -355,6 +355,25 @@ describe('src/cy/commands/assertions', () => {
 
         cy.contains('Nested Find').should('have.length', 2)
       })
+
+      it('can chain contains assertions off of cy.contains', () => {
```

Comment:
need to reference the regression/issue that this test is fixing

Reply:
> There are not 3 variants, there are only 2 right?

right